lockfile.json

```diff
@@ -782,6 +782,9 @@
   "vue": {
     "revision": "22bdfa6c9fc0f5ffa44c6e938ec46869ac8a99ff"
   },
+  "wit": {
```

Comment:
wrong place

Reply:
Thanks, resolved in 70648a1cc7dcbe69ce291a602742b94fe2a15609